<launch>

   <!-- Launch Arguments -->

   <!-- Driving Wheel Node -->
   <node pkg="teleop_control"
         type="teleop_driving_wheel_device"
         name="teleop_driving_wheel_device">

      <param name="axis_steering" value="0" type="int"/>
      <param name="axis_brake_pedal" value="3" type="int"/>
      <param name="axis_throttle" value="2" type="int"/>
      <param name="axis_deadman" value="1" type="int"/>

   </node>

</launch>
